.github/workflows/pypi_publish.yml

```diff
@@ -48,8 +59,122 @@ jobs:
           name: wheels-sdist
           path: ./wheelhouse/*.tar.gz
 
+  build-wheels:
+    name: 'Wheels: ${{ matrix.os }} ${{ matrix.arch }} ${{ matrix.python-version }}'
+    runs-on: ${{ matrix.os }}
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
```

Comment:
I think this line is redundant with `include`?

Reply:
I'm not sure why, but even with `fail-fast: false` and `continue-on-error: true`, only a subset of jobs are launched if I omit this. I'm not sure if that's because if some jobs fail, others don't bother starting (although I would expect that not to be the case from the options above...), or whether there is some other problem with specifying matrices this way. Either way, specifying the matrix as above seems to launch all jobs simultaneously, so I'll keep this as is for now.